src/tests.rs

```diff
@@ -1151,7 +1200,7 @@ declare namespace RootNs {
           "location": {
             "filename": "file:///test.ts",
             "line": 4,
-            "col": 4
+            "col": 18
```

Comment:
The variable declaration starts are now actually at the variable declaration starts instead of the variable statement.